README.md

```diff
@@ -0,0 +1 @@
+docs/README.md
```

Comment:
Update: This breaks the links to HACKING and INSTALL that are in the README document. They point to HACKING and INSTALL, since they're in the same directory as README.

Simpler: README is at the root level, and all the other documents are in docs.